src/pages/student/Bookmarks.css

```
@@ -41,6 +41,20 @@
     width: 300px;
 }
 
+.dashboard-events-photo-blob {
+    position: absolute;
+    width: 100%;
+    z-index: -1;
+    margin-left: 150px;
+    top: -60px;
+}
+
+.dashboard-blobblue-img {
```

This should be renamed for the bookmarks page

```suggestion
.bookmarks-blobblue-img {
```